src/ui/components/Control/SegmentedControl.tsx

```diff
@@ -0,0 +1,36 @@
+import SegmentedControlIOS from '@react-native-community/segmented-control';
+import React from 'react';
+
+interface SegmentedControlProps {
+  values?: any;
+  selectedIndex?: number;
+  style?: any;
+  momentary?: any;
+  enabled?: any;
+  tintColor?: any;
+  backgroundColor?: any;
+}
+
+const SegmentedControl: React.FC<SegmentedControlProps> = ({
+  values,
+  selectedIndex,
+  style,
+  momentary,
+  enabled,
+  tintColor,
+  backgroundColor
+}) => {
+  return (
+    <SegmentedControlIOS
+      values={values}
+      selectedIndex={selectedIndex}
+      style={style}
+      momentary={momentary}
+      enabled={enabled}
+      tintColor={tintColor}
+      backgroundColor={backgroundColor}
+    ></SegmentedControlIOS>
+  );
+};
+
+export default SegmentedControl;
```

Comment:
@deborahdiazr I'll let you make the call on this feedback:

`import SegmentedControlIOS from '@react-native-community/segmented-control';` is an iOS only component. Do you think that we should have a cross platform version of this pattern instead of or in addition to this one? I would like to maintain as few components as possible. There is a really nice version that you can grab from `beekman-society-oppy` and restyle in the theme of legos if you decide to go with the cross platform route. Great job on this stuff!